--- pandoc-types-1.20/pandoc-types.cabal~	2019-11-12 17:32:13.000000000 +0100
+++ pandoc-types-1.20/pandoc-types.cabal	2020-05-29 23:47:58.150522985 +0200
@@ -58,7 +58,7 @@
                      bytestring >= 0.9 && < 0.11,
                      aeson >= 0.6.2 && < 1.5,
                      transformers >= 0.2 && < 0.6,
-                     QuickCheck >= 2.4 && < 2.14
+                     QuickCheck
   if !impl(ghc >= 8.0)
     Build-depends:   semigroups == 0.18.*
   ghc-options:       -Wall
@@ -77,7 +77,7 @@
                        test-framework >= 0.3 && < 0.9,
                        test-framework-hunit >= 0.2 && < 0.4,
                        test-framework-quickcheck2 >= 0.2.9 && < 0.4,
-                       QuickCheck >= 2.4 && < 2.14,
+                       QuickCheck,
                        HUnit >= 1.2 && < 1.7,
                        string-qq >= 0.0.2 && < 0.1
   ghc-options:         -threaded -rtsopts -with-rtsopts=-N -Wall -O2
